graphitron-codegen-parent/graphitron-java-codegen/src/test/java/no/sikt/graphitron/resolvers/datafetchers/standard/fetch/StrategyNodeResolverTest.java

```diff
@@ -63,15 +63,21 @@ void nodeOneImplementation() {
         assertGeneratedContentMatches("oneImplementation");
     }
 
+    @Test
+    @DisplayName("One type with node directive generates correct switch statement")
+    void nodeCorrectSwitchStatement() {
+        assertGeneratedContentContains("oneImplementation", "case \"CustomerType\"");
```

Comment:
Kan du ta bort endringen du har i `graphitron-codegen-parent/graphitron-java-codegen/src/test/resources/resolvers/datafetchers/fetch/strategynode/oneImplementation/schema.graphqls` og så lage en egen skjemafil for testen her?
F.eks.
```suggestion
        assertGeneratedContentContains("withCustomTypeId", "case \"CustomerType\"");
```